azext_iot/digitaltwins/params.py

```diff
@@ -525,3 +525,57 @@ def load_digitaltwins_arguments(self, _):
             options_list=['--yes', '-y'],
             help='Do not prompt for confirmation when assigning required roles.',
         )
+
+    with self.argument_context("dt job import") as context:
+        context.argument(
+            "job_id",
+            options_list=["--job-id", "-j"],
+            help="ID of bulk import job",
+            arg_group="Bulk Import Job",
+        )
+
+        context.argument(
+            "data_file_name",
+            options_list=["--data-file", "--df"],
+            help="Name of data file input to the bulk import job. The file must be in 'ndjson' format. Sample input data "
+            "file: https://github.com/Azure/azure-iot-cli-extension/tree/dev/docs/samples/adt-bulk-import-data-sample.ndjson",
+            arg_group="Bulk Import Job",
+        )
+
+        context.argument(
+            "input_blob_container_name",
+            options_list=["--input-blob-container", "--ibc"],
+            help="Name of Azure Storage blob container which stores the bulk import data file.",
```

Comment:
contains may be better here than stores to show that the data file is in that container already